compliance_suite/web/partials/navbar.html

```diff
@@ -1,3 +1,6 @@
 <nav id="navbar" class="navbar navbar-expand-lg navbar-light bg-light">
-    <a id="navbar-title" class="navbar-brand" href="#">GA4GH TES v0.1.0 Compliance Report</a>
+    <a id="navbar-title" class="navbar-brand" href="#">GA4GH TES Compliance Report</a>
+    {% for version in versions %}
+        <a id="navbar-title" class="navbar-brand" href="report-tes-{{version}}.html">TES v{{version}}</a>
+    {% endfor %}
```

Comment:
I think this is good opportunity to make the test runner more generic by not only not hardcoding the version, but also not hardcoding the API _name_ (here: "GA4GH TES"). Also, "Compliance" is perhaps too specific, I would rather go with "Test Report".

Reply:
I agree, @lbeckman314 added a new field `name`. That can be used here.